pos_customer_required/models/pos_config.py

```diff
@@ -12,15 +12,11 @@ class PosConfig(models.Model):
     _inherit = 'pos.config'
 
     _REQUIRE_CUSTOMER_KEYS = [
-        ('no', 'Optional'),
```

Comment:
The lack of this is breaking your tests

Reply:
Hi. I have no time to fix that point, but feel free to make commit on that PR to fix that problem.

Reply:
:smile: @legalsylvain we are not using POS in v9 :thinking: we also cannot take time here as we are working hard en v11 :+1:  we just wanted to help and have v9 branch green